cms/release_calendar/models.py

```diff
@@ -68,6 +69,7 @@ class ReleaseCalendarPage(BasePage):  # type: ignore[django-manager-missing]
     )
 
     content = StreamField(ReleaseCalendarStoryBlock(), blank=True)
+    datasets = StreamField(DatasetStoryBlock(), blank=True, use_json_field=True)
```

Comment:
`use_json_field` is not necessary anymore

https://docs.wagtail.org/en/latest/releases/6.0.html#streamfield-no-longer-requires-use-json-field-true

Reply:
Addressed in https://github.com/ONSdigital/dis-wagtail/pull/123/commits/6ce4cf0620a7652c93c212421964b07cc5b5d86f